ui/pages/keychains/restore-vault.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import RestoreVaultPage from './restore-vault';
+
+const mockStore = configureStore([]);
+const store = mockStore({
+  appState: {
+    isLoading: false,
+  },
+});
+
+const meta: Meta<typeof RestoreVaultPage> = {
+  title: 'Pages/Keychains/RestoreVaultPage',
```

Comment:
It's excellent that Devin has picked up the convention that the title should echo the folder path 💯